care/emr/resources/scheduling/schedule/spec.py

```diff
@@ -123,10 +124,18 @@ class ScheduleCreateSpec(ScheduleBaseSpec):
     user: UUID4
     facility: UUID4
     name: str
-    valid_from: datetime.datetime
-    valid_to: datetime.datetime
+    valid_from: datetime.date
```

Comment:
Technically speaking we should support datetime here right ? like the availability might only start from Tue noon on the 10th ( They might have a slot on tuesday morning ) ? @rithviknishad

Reply:
oh yes